### ISSUE TITLE
S4 hibernate issue

### ISSUE THREAD

**Issue by gzliudianxin** [jkbuha/XPS-9500-IceLake-OpenCore#10]
Hello, xps9500 10300H, using your same EFI file。
When I enter s4 sleep mode, after a period of about half an hour, the system will automatically restart once and then repeatedly cycle this action. 
When I sleep for a long time, after a night, when I turn on,the page displaying DELL's logo and get stuck 。I can only start by forcibly shutting down and restarting. 
All settings have been executed according to your tutorial steps, including setting BIOS options and unlocking CFG. And executed the pmset command to set the sleep mode. 
Would you encounter the same problem?

**Comment by jkbuha** [comment]
I suspect it’s an issue with powernap settings.

Can you try this and see if it works?

sudo pmset -a powernap 0

**Comment by gzliudianxin** [comment]
I set the powernap according to your instructions, and conducted a long period of testing。
The restart phenomenon has improved slightly, but it still occasionally occurs. 
Here is my pmset settings：
------------------------------------------
pmset -g custom                                       
Battery Power:
 lidwake              1
 lowpowermode         0
 standbydelayhigh     1
 proximitywake        0
 standby              1
 standbydelaylow      1
 ttyskeepawake        1
 highstandbythreshold 50
 powernap             0
 hibernatemode        25
 hibernatefile        /var/vm/sleepimage
 displaysleep         10
 sleep                1
 tcpkeepalive         1
 halfdim              1
 acwake               0
 lessbright           1
 disksleep            10
AC Power:
 lidwake              1
 lowpowermode         0
 standbydelayhigh     1
 standbydelaylow      1
 standby              1
 proximitywake        0
 ttyskeepawake        1
 hibernatemode        25
 powernap             0
 highstandbythreshold 50
 hibernatefile        /var/vm/sleepimage
 displaysleep         20
 womp                 1
 networkoversleep     0
 sleep                1
 tcpkeepalive         1
 halfdim              1
 acwake               0
 disksleep            20
----------------------------
I tried using the command to check the reason for wake-up and found the following logs. 
Can I do anything to solve this problem?
<img width="1121" alt="截屏2024-05-03 09 02 30" src="https://github.com/jkbuha/XPS-9500-IceLake-OpenCore/assets/161866307/f82ab814-e6cf-4f15-bd89-92e628b978ff">
